Directory.Build.targets

```
@@ -34,4 +44,26 @@
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(IsTestProject)' != 'true' and '$(ExcludeFromAssemblyMetadata)' != 'true'" >
+    <AssemblyAttribute Include="System.Reflection.AssemblyDefaultAliasAttribute">
+      <_Parameter1>$(AssemblyName)</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Reflection.AssemblyDescriptionAttribute">
+      <_Parameter1>$(AssemblyName)</_Parameter1>
+    </AssemblyAttribute>
+    <AssemblyAttribute Include="System.Runtime.InteropServices.DefaultDllImportSearchPathsAttribute">
```

I would condition this on the project being a csproj:

```suggestion
    <AssemblyAttribute Condition="'$(MSBuildProjectExtension)' == '.csproj'" Include="System.Runtime.InteropServices.DefaultDllImportSearchPathsAttribute">
```

---

Why would we need this?

---

That assembly attribute with the below syntax only works for C# assemblies.